content/events/2021-online-unconference/_index.md

```diff
@@ -89,3 +89,22 @@ We will publish submitted contributions here as soon as they are confirmed.
   before, during, and after the event
   
  
+## Share the event
+
+Do you want to share the event with your colleagues/friends? Here are a few texts that you can reuse, feel also free to modify them as you like
+
+### Short teaser
+#NordicRSE invites everyone interested in "Research Software Engineering" activities to join & shape the agenda of the NordicRSE unconference (lightweight get-together) on June 29-30. See [https://nordic-rse.org/events/2021-online-unconference/](https://nordic-rse.org/events/2021-online-unconference/) for more details #RSEng @EOSC_Nordic @coderefine @SciCompAalto
+
+### Longer teaser
+
+* Are you developing software or tools that are driven by research/engineering in either academia or industry?
+* Need to network, share knowledge and experiences with your peers? 
+* Maybe you have heard of something called research software engineers and you would like to know more? 
+   
+#NordicRSE invites everyone interested in such topics to join our online conference (lightweight get-together) on June 29 and 30, where we let the participants shape the agenda ("birds of a feather"). To kick off the event, we also have four invited speakers, including:
+* one of the Julia programming language core developers, 
+* an R-developer working a an Research Software Engineer (RSE), 
+* one of the developers behind QuTiP (Quantum Toolbox in Python).
```

Comment:
any particular reason for not having the names? These three are confirmed right?

Reply:
Yes they are confirmed. By the time of writing, we were still waiting for their consent to publish names, titles/abstracts. Now we have the consent for all three!

Reply:
should we add the names (and possibly link to their homepage) here?

Reply:
Good ideas! Added suggestions for updates.